src/Agent/MyProfile.vue

```diff
@@ -4,7 +4,8 @@
   <b-collapse v-model="showCollapse" id="collapseAgentInfo" class="mt-2">
     <b-row>
       <b-col cols="4">
-        <b-img src='src/assets/default_agent.png' style="width:100%" alt="avatar image" />
+        <b-img v-if="agent.avatar != 'undefined'" :src="$agent.get_rr_uri()+'/avatar/'+agent.avatar" style="width:64px;"/>
```

Comment:
imo better to place it to function: $agent.get_rr_uri()+'/avatar, e.g. avatar_uri()